phc/easy/query/fhir_dsl.py

```diff
@@ -19,12 +28,54 @@ def query_allows_scrolling(query):
 
 
 def execute_single_fhir_dsl(
-    query: dict, scroll_id: str = "", auth_args: Auth = Auth.shared()
+    query: dict,
+    scroll_id: str = "",
+    retry_backoff: bool = False,
+    auth_args: Auth = Auth.shared(),
+    _retry_time: int = 1,
 ):
     auth = Auth(auth_args)
     fhir = Fhir(auth.session())
 
-    return fhir.execute_es(auth.project_id, query, scroll_id)
+    if _retry_time >= MAX_RETRY_BACKOFF or retry_backoff is False:
```

Comment:
Can we move this into the `try-catch` as part of the check on line 46? That way we only have one path for executing `fhir.dsl`.